### PR TITLE
[SRVKE-1579] Customizing Kafka topic config options

### PR DESCRIPTION
**Pull request by ochromy**
Version(s):
serverless-docs-1.33+

Issue:
https://issues.redhat.com/browse/SRVKE-1579

Link to docs preview:
- https://82156--ocpdocs-pr.netlify.app/openshift-serverless/latest/eventing/brokers/kafka-broker.html#serverless-kafka-broker-configmap_kafka-broker

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mgencur** [comment]
LGTM from QE

File: modules/serverless-kafka-broker-configmap.adoc [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 You can configure the replication factor, bootstrap servers, and the number of topic partitions for a Kafka broker, by creating a config map and referencing this config map in the Kafka `Broker` object.
 
+Knative Eventing supports the full set of topic config options that Kafka supports. To set any of these, you need to add a key to the configmap with the `default.topic.config.` prefix.
```
Comment (xenolinux) [inline modules/serverless-kafka-broker-configmap.adoc:11]:
```suggestion
Knative Eventing supports the full set of topic config options that Kafka supports. To set these options, you must add a key to the ConfigMap with the `default.topic.config.` prefix.
```

**Review by xenolinux — COMMENTED** [review]
Tiny nits; otherwise LGTM

File: modules/serverless-kafka-broker-configmap.adoc [hunk 2]
```diff
@@ -30,12 +32,14 @@ data:
...
   default.topic.replication.factor: <integer> <4>
   bootstrap.servers: <list_of_servers> <5>
+  default.topic.config.<config_option>: <value> <6>
 ----
 <1> The config map name.
 <2> The namespace where the config map exists.
 <3> The number of topic partitions for the Kafka broker. This controls how quickly events can be sent to the broker. A higher number of partitions requires greater compute resources.
 <4> The replication factor of topic messages. This prevents against data loss. A higher replication factor requires greater compute resources and more storage.
 <5> A comma separated list of bootstrap servers. This can be inside or outside of the {ocp-product-title} cluster, and is a list of Kafka clusters that the broker receives events from and sends events to.
+<6> A topic config option. See the link:https://kafka.apache.org/documentation/#topicconfigs[full set of possible options and values].
```
Comment (xenolinux) [inline modules/serverless-kafka-broker-configmap.adoc:42]:
```suggestion
<6> A topic config option. For more information, see the link:https://kafka.apache.org/documentation/#topicconfigs[full set of possible options and values]. 
```

**Pull request merged**